packages/components/src/icons/logos/Paypal.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+
+export const PaypalLogoIcon = (): JSX.Element => {
+  return (
+    <svg
+      width="64px"
+      height="16px"
+      viewBox="0 0 64 16"
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+    >
+      <g id="Symbols" stroke="none" stroke-width="1" fill="none" fillRule="evenodd">
+        <g id="Donation" transform="translate(-43.000000, -132.000000)">
```

Comment:
You should strip out ids from svgs and also make sure you fix `stroke-width`

Reply:
Good call, cleaned up!